package.json

```diff
@@ -11,9 +11,11 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "ajv": "^8.11.0",
+    "aws-sdk": "^2.1362.0",
     "bootstrap": "^3.4.1",
     "bootstrap-loader": "^4.0.0",
     "bootstrap-sass": "^3.4.3",
+    "dotenv": "^16.0.3",
```

Comment:
When the app is deployed to GCP, it won't be using a .env file, the env variables will be set explicitly, so you can get rid of this dependency also.